docs/architecture/architecture-overview.adoc

```diff
@@ -0,0 +1,244 @@
+      --namespace string   --namespace=myproject
+  -o, --output string      -o=json -o=template (default "table")
+ Use "mobile [command] --help" for more information about a command.
+ MacBook-Pro:images joeBloggs$
+
+There is a separate repo for the Mobile CLI. Setup and usage guides are available https://github.com/aerogear/mobile-cli[here].
+
+ Note; currently the Mobile CLI is not installed during MC startup, but the intention is to add it to the installation process in the future.
+
+Once installed, there is no live/running MC process/server. There is no openshift pod for MC. Once MC is installed and configured its functionality is available throughout all Openshift projects/namespaces.
+
+== MC Functionality
+=== Create a representation of a Mobile App
+The MC UI extension provides the user with the capabilitiy to create a representation of a Mobile App inside their Openshift project.
+
+MC allows the user to select the Mobile filter in the Service Catalog and select a representation of an App type to create. Note; an App representation is a set of basic properties for the App. It is not a running process. Currently its a visual represenation of an App.
+
+image::images/5.x-ServiceCatalog-MobileApps.png[width=1280, title="Service Catalog - Mobile App Perspective", link="images/5.x-ServiceCatalog-MobileApps.png"]
+
+When this App representation is created, an instance of the associated CRD is created, see Example below.
+
+
+.Example of an instantiated CRD for an Android Mobile App Representation
+====
+ MacBook-Pro:architecture user1$ oc get mobileclients -o=json
+ {
+     "apiVersion": "v1",
+     "items": [
+         {
+             "apiVersion": "mobile.k8s.io/v1alpha1",
+             "kind": "MobileClient",
+             "metadata": {
+                 "clusterName": "",
+                 "creationTimestamp": "2018-01-04T12:35:12Z",
+                 "deletionGracePeriodSeconds": null,
+                 "deletionTimestamp": null,
+                 "labels": {
+                     "icon": "fa-android"
+                 },
+                 "name": "my.app-1515069311",
+                 "namespace": "myproject",
+                 "resourceVersion": "17980",
+                 "selfLink": "/apis/mobile.k8s.io/v1alpha1/namespaces/myproject/mobileclients/my.app-1515069311",
+                 "uid": "b5840229-f14b-11e7-95e5-b242a86257d4"
+             },
+             "spec": {
+                 "apiKey": "3e88bede-0c80-4df1-8cd9-cf9b5f50b771",
+                 "clientType": "android",
+                 "name": "my.app"
+             }
+         }
+     ],
+     "kind": "List",
+     "metadata": {
+         "resourceVersion": "",
+         "selfLink": ""
+     }
+ }
+====
+Further details on creating a custom resource object (instantiation of the CRD) is detailed https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/[here].
+
+=== Instantiate a Mobile Service
```

Comment:
Would 'Provision a Mobile Service' sound better?
I'm not sure what the correct terminology in OpenShift or the Catalog is.
I've heard 'Order' in the context of the Catalog. e.g. Order a Service.